addons/cni-cilium/cilium.yaml

```diff
@@ -7,7 +7,7 @@
 #   - templated cluster-pool-ipv4-cidr
 #   - templated kube-proxy-replacement parts
 #   - added seccomp profile to cilium-operator
-
+{{ $hubble_ipv6 := default "true" .Params.HubbleIPv6 }}
```

Comment:
seems like the name of this parameter deserves to be documented in the README

Reply:
@kron4eg Good point, added, PTAL.